src/components/Input/Input.styles.jsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export const StyledInput = styled.input`
+	display: block;
+	width: 100%;
+	font-family: "Mulish", sans-serif;
```

Comment:
Myślę, że jeśli wszystkie elementy mają mieć tego fonta, to warto ustawić to poprawnie w global styles ;)